scripts/lint_preprocessor_deps.sh

```diff
@@ -1,16 +1,26 @@
 #!/bin/bash
 
+set -eou pipefail
```

Comment:
I get the `set -euo pipefail` as good practice but seems like you can remove the `-u` here, the `+u` below and keep the last `-u` if you'd like (though no variables expanded following could be unset).

Reply:
Hmm, I thought about this more -- I disagree on this one. Yes, it will work, but if code changes around we may forget to enable the `-u` later. For consistency sake, I want to keep this here.

Feel free to open an issue or DM me and we can discuss more, though!